registration/app/registration/controllers/Main.scala

```diff
@@ -135,4 +139,14 @@ final class Main(
       InternalServerError(other.reason)
   }
 
```

Comment:
I'd like to, it would be good to, and started doing it, but it depends on quite a few things in this class